tasks.py

```diff
@@ -423,11 +423,12 @@ def bgp_dev_env(ip_family):
     with open("%s/bgpd.conf" % frr_volume_dir, 'w') as f:
         f.write(bgpd_config)
 
+    run("echo '/etc/frr/core-%e.%p.%h.%t' | sudo tee /proc/sys/kernel/core_pattern")
```

Comment:
Same, we don't know where this will be running and we don't always have root permissions.